src/sentry/utils/files.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
Not sure if this is overkill or not, but I wanted to compress iteratively while reading the file stream, as well as maintain the un-compressed version.

Alternatively, could be simplified by just doing:

```python
body = fp.read()
z_body = zlib.compress(body)
```

instead and avoid this dance. I don't know if it particularly matters.